apps/labs/posts/numpy-python-api-cleanup.md

```diff
@@ -0,0 +1,345 @@
+---
+title: 'Python API cleanup for NumPy 2.0'
+published: October 11, 2023
+author: mateusz-sokol
+description: 'A journey through NumPy's Python API from a maintenance perspective.'
+category: [NumPy]
+featuredImage:
+  src: /posts/numpy-python-api-cleanup/numpy-python-api-cleanup-featured.png
```

Comment:
Looks like you need to decide where you want a stock image, or something else.

Reply:
I updated images